core/src/test/scala/org/apache/spark/storage/DiskStoreSuite.scala

```diff
@@ -22,10 +22,12 @@ import java.util.Arrays
 
 import org.apache.spark.{SparkConf, SparkFunSuite}
 import org.apache.spark.util.io.ChunkedByteBuffer
+import org.apache.spark.util.Utils
 
 class DiskStoreSuite extends SparkFunSuite {
 
   test("reads of memory-mapped and non memory-mapped files are equivalent") {
+    assume(!Utils.isWindows)
```

Comment:
It will cause error when we tried to re-open the filestore and the memory-mapped byte buffer to the file has not been GC. 

Ref: http://bugs.java.com/bugdatabase/view_bug.do?bug_id=4724038
http://stackoverflow.com/questions/3602783/file-access-synchronized-on-java-object